libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
A wrapper cookbook would never manage rabbitmq and other services? I find this hard to believe as there are cookbooks like sensu that manage RMQ, redis, etc.

Would it help if we made it more like `remove_self_from_rabbitmq_current_cluster` ?